cmd/head-main.go

```diff
@@ -80,11 +78,10 @@ EXAMPLES:
      {{.Prompt}} {{.HelpName}} -n 1 s3/csv-data/population.csv.gz
 
   2. Display only first line from server encrypted object on Amazon S3.
-     {{.Prompt}} {{.HelpName}} -n 1 --encrypt-key 's3/csv-data=32byteslongsecretkeymustbegiven1' s3/csv-data/population.csv
+     {{.Prompt}} {{.HelpName}} -n 1 --enc-c 's3/csv-data=MDEyMzQ1Njc4OTAxMjM0NTY3ODkwMTIzNDU2Nzg5MDA' s3/csv-data/population.csv
```

Comment:
Funny. I have the same key on my luggage...

Reply:
0 through 9 never fails